### PR TITLE
Update publications.json

### PR DESCRIPTION
**Pull request by jiafei1224**
Added AR2-D2 paper

### CODE REVIEW

**Comment by mattdeitke** [comment]
Thanks @jiafei1224!

Are you missing a file for the thumnail? I don't see static/ar2d2.jpg uploaded

If there's no thumbnail, that line can be removed.

**Comment by mattdeitke** [comment]
also, I believe this paper is already listed: https://github.com/RAIVNLab/RAIVNLab.github.io/blob/main/src/components/publications.json#L291-L304

File: src/components/publications.json [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+      "Yi Ru Wang",
+      "Mohit Shridhar",
+      "Dieter Fox",
+      "Ranjay Krishna"
+    ],
+    "year": "2023",
+    "venue": "CoRL 2023",
+    "links": {
+      "pdf": "https://arxiv.org/pdf/2306.13818.pdf",
+      "Project page": "ar2d2.site"
```
Comment (mattdeitke) [inline src/components/publications.json:15]: can you make this `"project page"` (lowercase p). Otherwise, it will not work.

Comment (jiafei1224) [reply src/components/publications.json:15]: I added the lowercase for the project in another pull request. Thanks Matt

Comment (mattdeitke) [reply src/components/publications.json:15]: Can you edit the existing entry instead of adding a new one? The paper is already listed:
image

Comment (jiafei1224) [reply src/components/publications.json:15]: Ohh yes, i make another PR, to add in the thumbnail.

Comment (mattdeitke) [reply src/components/publications.json:15]: merged the new PRs - will take a few minutes for the build to complete :)

Closing this

**Pull request closed without merging**